config/v1/types_console.go

```diff
@@ -60,3 +63,29 @@ type ConsoleAuthentication struct {
 	// +kubebuilder:validation:Pattern=`^$|^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))$`
 	LogoutRedirect string `json:"logoutRedirect,omitempty"`
 }
+
+// Helm configuration
```

Comment:
Why Helm specifically, why not creating something more generic and add a helm type for starters?

Reply:
umm.. I see what you are saying, do you imply something like :

```
type Registry struct {
	URL string `json:"url"`
	Name string `json:"name"`
	Description string `json:"description"`
}

..
..

type ConsoleSpec struct {
	// +optional
	Authentication ConsoleAuthentication `json:"authentication"`

	// +optional
	Helm *[]Registry `json:"helm"`
}

```

and then `Registry` could be used to configure _any_ external registry/repository that console needs to be aware of in future.